roles/postgresql_upgrade/molecule/redhat/prepare.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Prepare
+  hosts: all
+  gather_facts: true
+  roles:
+    - puppet_repositories
+    - foreman_repositories
+    - role: installer
+      vars:
+        installer_scenario: foreman
```

Comment:
should this explicitly point at a foreman version that came with psql 10, as we plan to shift the default to 12 soon?

Reply:
Good point, I'll point it at 2.3.